rero/rero-ils-ui
# acquisition: implement order views

* Adds order and order line brief view.
* Adds order and order line detail view.
* Adds link to acquisition menu.

Co-Authored-by: Laurent Dubois <laurent.dubois@itld-solutions.be>

## Why are you opening this PR?

- Implement acquisition order and order line.

## How to test?

- Use rero/rero-ils/pull/689 to test.
- Check Order in Acquisition menu

## Code review check list
